packages/infero/package.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+#
+from spack import *
+
+
+class Infero(CMakePackage):
+    '''
+    Infero runs a pre-trained ML model for inference. It can be deployed on a HPC system 
+    without the need for high-level python libraries (e.g. TensorFlow, PyTorch, etc..)
+    '''
+
+    ecmwf = "https://github.com/ecmwf-projects/infero.git"
+
+    version('0.1.2',
+            git=ecmwf,
+            commit='4c229a16ce75a249c83cbf43e0c953a7a42f2f83')
+
+    maintainers = ['juckerj']
+
+    variant('quiet', description='Disable calls to Log::info', default=False)
+
+    depends_on('eckit@1.20.0:')
```

Comment:
Why is there a colon at the end of `eckit@1.20.0`? Is this a placeholder?

Reply:
`1.20.0:` is spack syntax for version 1.20.0 and up.
`:1.20.0` means versions up to 1.20.0 and `1.19.0:1.20.0` means all versions between 19 and 20. Version ranges in Spack are inclusive. https://spack.readthedocs.io/en/latest/packaging_guide.html#version-ranges